code/game/mecha/mech_fabricator.dm

```diff
@@ -135,7 +150,35 @@
 			update_static_data(user)
 		return
 	return ..()
-
+/**
+ * All the negative wire effect
+ * Zap doesnt actually zap, it breaks one limb (Because pain is to be had)
+*/
+/obj/machinery/mecha_part_fabricator/proc/wire_zap(mob/user)
+	if(stat & (BROKEN|NOPOWER))
+		return FALSE
+	var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread
+	s.set_up(5, 1, src)
+	s.start()
+	var/mob/living/carbon/C = user
+	var/datum/wound/blunt/severe/break_it = new
+	///Picks limb to break. People with less limbs have a chance of it grapping at air
+	var/picked_bodypart = pick(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)
+	var/obj/item/bodypart/bone = C.get_bodypart(picked_bodypart)
+	if(bone){
```

Comment:
no braces

Reply:
Ok can I get elaboration on this? I have optimised it to
`var/obj/item/bodypart/bone = C.get_bodypart(pick(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG))`
to lower the memory impact but it still isnt in any sort of brackets itself.

Reply:
Ah I see I am blind